lib/utils/fsHelper.js

```diff
@@ -0,0 +1,30 @@
+const fs = require('fs');
```

Comment:
The latest file in `pharmacy-data-etl` has a changes in for the output directory coming from config.